coalib/output/ConsoleInteraction.py

```diff
@@ -548,20 +548,17 @@ def choose_action(console_printer, actions):
     :param actions:         Actions available to the user.
     :return:                Return choice of action of user.
     """
-    console_printer.print(format_lines(
-        "The following actions are applicable to this result:"))
-
     while True:
         console_printer.print(format_lines("*0: " +
-                                           "Apply no further actions."))
+                                           "No further actions."))
```

Comment:
"Do nothing" would be even shorter, I think even being a bit shorter helps people to read this so I'd also remove the period